single-line-diagram-server/src/main/java/com/powsybl/single/line/diagram/server/SingleLineDiagramController.java

```diff
@@ -0,0 +1,59 @@
+package com.powsybl.single.line.diagram.server;
```

Comment:
Package is not correct, see discussion about package naming good practices: https://github.com/powsybl/powsybl-core/pull/983
=> com.powsybl.sld.server?